x-pack/platform/plugins/shared/lens/public/react_embeddable/initializers/initialize_dashboard_services.ts

```diff
@@ -78,9 +78,7 @@ export function initializeDashboardServices(
 ): DashboardServicesConfig {
   // For some legacy reason the title and description default value is picked differently
   // ( based on existing FTR tests ).
-  const defaultTitle$ = new BehaviorSubject<string | undefined>(
-    initialState.title || internalApi.attributes$.getValue().title
-  );
+  const defaultTitle$ = new BehaviorSubject<string | undefined>(initialState.attributes.title);
```

Comment:
This is great! The `defaultTitle` should only ever come from a by-reference Lens embeddable and those properties are formed on the `initialState.attributes` object. The `internalApi.attributes` are based on LensRuntimeState which is state stored on a dashboard and those properties should not be used to set the `defaultTitle`.